src/fsharp/service/ServiceXmlDocParser.fs

```diff
@@ -187,4 +188,13 @@ module XmlDocParser =
     /// Get the list of Xml documentation from current source code
     let getXmlDocables (sourceCodeOfTheFile, input) =
         let sourceCodeLinesOfTheFile = String.getLines sourceCodeOfTheFile
+        XmlDocParsing.getXmlDocablesImpl (sourceCodeLinesOfTheFile, input)
+
+    /// Get the list of Xml documentation from current source code
+    let getXmlDocablesFromSourceText (sourceText: ISourceText, input) =
```

Comment:
I wasn't concerned with the allocation problem. It was more just trying to get the information from SourceText directly since the other function `getXmlDocables` needed the full source string